README.md

```diff
@@ -9,16 +9,13 @@ This repository hosts the CSI KubeVirt driver and all of its build and dependent
 
 ## Deployment
 //TODO WIP
-- create a service-account with name on the kubevirt cluster (infra-cluster)
-- set RBAC rules for the service account (//TODO supply file, doc)
-- designate a namespace to deploy the VMs for the tenant cluster nodes
-- deploy tenant cluster
-- deploy files under `deploy/*`
-- edit the secret `infra-cluster-credentials` in namespace `kubevirt-csi-driver`
-    - set apiUrl: [base64 of https://<infra-cluster-api-url>]  
-    - set service-ca.crt : [base64 of <infra-cluster serivce-ca.crt>] (copy service-ca.crt value from infra-cluster-sa secret)
-    - set namespace: [base64 of <infra-cluster namespace>]
-    - set token : [base64 of <infra-cluser token>] (copy token value from infra-cluster-sa secret )
+- use `deploy/infra-cluster-service-account.yaml` to create a service account in kubevirt cluster (replace INFRA_NAMESPACE with the kubevirt cluster namepsace)
```

Comment:
another option for the namespace, is to omit it from the yaml and let the creator do it on its currently logged in namespace. More tool friendly, what do you think?

Reply:
I think that the creator will be a cluster admin in the infra. Therefore, I think that the logged in namespace might not be relvant.

Reply:
I see that create command can be used with '-n' flag. I can remove the namespace form the YAML and let user specify it via command line

Reply:
I'm thinking about automation mostly, but it seems that there is no way in kubectl or oc to specify a namespace flag for `create`

Reply:
Actually I just tried it and one can do `oc create -f x.yaml -n mynamespace` 
but only if namespace is not defined on the yaml. I wonder if kubectl behaves the same way.